src/core/public/i18n/i18n_eui_mapping.tsx

```diff
@@ -966,7 +966,7 @@ export const getEuiContextMapping = (): EuiTokensObject => {
     ),
     'euiSelectable.searchResults': ({ resultsLength }: EuiValues) =>
       i18n.translate('core.euiSelectable.searchResults', {
-        defaultMessage: '{resultsLength, plural, one {# result} other {# results}}',
+        defaultMessage: '{resultsLength, plural, one {# result} other {# results}} available',
```

Comment:
The new tests caught this default message mistake from a few upgrades ago. Hooray! :)